docs/hardware/srobo/mbv4.rst

```diff
@@ -1,7 +1,115 @@
+
+USB interface
+-------------
+
+Unlike other SR v4 boards, the STM32 on the motor board does not directly communicate over USB. Instead there is an FTDI USB serial interface chip.
+
+The FTDI Chip has vendor ID ``0403`` and product ID ``6001``. It can be further filtered by the USB ``product`` field.
+
+Given that the appropriate drivers are available on your computer, it should appear as a standard serial interface. You should open this interface at a baud rate of ``1000000bps``.
+
+Serial Commands
+~~~~~~~~~~~~~~~
+
+8 bit commands should be sent over the serial interface to the board.
+
+The following commands are supported:
+
++---------------+-------------------+---------------------------+
+| Decimal Value | Unicode           | Purpose                   |
++===============+===================+===========================+
+| 0             | ``\x00``          | Reset the STM32           |
++---------------+-------------------+---------------------------+
+| 1             | ``\x01``          | Send the firmware string  |
++---------------+-------------------+---------------------------+
+| 2             | ``\x02``          | Set Motor 0 value         |
++---------------+-------------------+---------------------------+
+| 3             | ``\x03``          | Set Motor 1 value         |
++---------------+-------------------+---------------------------+
+| 4             | ``\x04``          | Reset into the Bootloader |
++---------------+-------------------+---------------------------+
+
+
+Setting Motor Speeds
+~~~~~~~~~~~~~~~~~~~~
+
+The value of a motor can be set by sending the appropriate command, and then a second byte as follows:
+
++---------------+-------------------------------------------------------------+
+| Decimal Value | Purpose                                                     |
++===============+=============================================================+
+| 0             | Do nothing                                                  |
++---------------+-------------------------------------------------------------+
+| 1             | Coast the motor                                             |
++---------------+-------------------------------------------------------------+
+| 2             | Brake the motor                                             |
++---------------+-------------------------------------------------------------+
+| 3 - 127       | Set speed of motor to negative.                             |
++---------------+-------------------------------------------------------------+
+| 128           | Set speed of motor to 0                                     |
++---------------+-------------------------------------------------------------+
+| 129 - 255     | Set speed of motor positive. Subtract 128 to get the value. |
++---------------+-------------------------------------------------------------+
```

Comment:
I think it would be more concise to simply say that the argument byte is a standard 8-bit signed number, where a few values are reserved for the special cases of no-op, coast and brake.

Reply:
8ea296b